phpstan-baseline.neon

```diff
@@ -71625,3 +71625,65 @@ parameters:
 			message: "#^Call to an undefined method MauticPlugin\\\\MauticTagManagerBundle\\\\Security\\\\Permissions\\\\TagManagerPermissions\\:\\:expects\\(\\)\\.$#"
 			count: 1
 			path: plugins/MauticTagManagerBundle/Tests/Unit/Security/Permissions/TagManagerPermissionsTest.php
+
+		- '#\$id type mapping mismatch: database can contain string but property expects int.#'
```

Comment:
@escopecz this is related casting of `bigint` in mysql to int in php, the details are mentioned in comments of this PR